test/intl402/RelativeTimeFormat/prototype/format/unit-invalid.js

```diff
@@ -0,0 +1,43 @@
+// Copyright 2018 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-Intl.RelativeTimeFormat.prototype.format
+description: Checks the handling of invalid unit arguments to Intl.RelativeTimeFormat.prototype.format().
+info: |
+    SingularRelativeTimeUnit ( unit )
+
+    10. If unit is not one of "second", "minute", "hour", "day", "week", "month", "quarter", "year", throw a RangeError exception.
+
+features: [Intl.RelativeTimeFormat]
+---*/
+
+const rtf = new Intl.RelativeTimeFormat("en-US");
+
+assert.sameValue(typeof rtf.format, "function");
```

Comment:
This is irrelevant to the semantics of `SingularRelativeTimeUnit(...)`. There is already a test file ([prop-desc.js](https://github.com/tc39/test262/blob/master/test/intl402/RelativeTimeFormat/prototype/format/prop-desc.js)) that verifies `Intl.RelativeTimeFormat.prototype.format` is function.